docs/src/pages/components/lists/DisabledGutters.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Avatar from '@material-ui/core/Avatar';
+import Card from '@material-ui/core/Card';
+import CardHeader from '@material-ui/core/CardHeader';
```

Comment:
Please remove all usage of the card component. We are focusing on the list in this page.

Reply:
Also, I think that we should aim simplifying the demo as much as possible. Less than 20 lines would be perfect.

Reply:
I'm wondering if it would be better to remove my demo and add a  `Disable gutters` checkbox to the interactive demo?

Reply:
Here's what that could look like:

![image](https://user-images.githubusercontent.com/3155127/87084791-ca68b300-c1fc-11ea-924c-2274855b06d5.png)

Note: The secondary action is currently using `edge="end"` which is pulling it out of the list.

Reply:
We are trying as much as possible not to rely on a playground to document the component. Usually, it's simple to search on the page and to not have to click on any buttons to see what you are looking for.

Reply:
But yeah, in this case, why not

Reply:
Your point about the playgrounds is a good one.  I'll follow your original guidance.  Thanks.